ci-operator/step-registry/assisted/common/setup/test/assisted-common-setup-test-commands.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+echo "************ assisted common setup test command ************"
+
+# Get packet | vsphere configuration
+# shellcheck source=/dev/null
+set +e
+source "${SHARED_DIR}/packet-conf.sh"
+source "$SHARED_DIR/ci-machine-config.sh"
+set -e
+
+# TODO: Remove once OpenShift CI will be upgraded to 4.2 (see https://access.redhat.com/articles/4859371)
+~/fix_uid.sh
+
+ssh -i ${SSH_KEY_FILE} -o ConnectTimeout=5 -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o ServerAliveInterval=90 -o LogLevel=ERROR root@${IP} "source /root/config.sh;  cd /home/assisted; make \${MAKEFILE_TARGET:-create_full_environment run test_parallel}"
```

Comment:
too long of a line, that the important part is a bit hidden
I'd suggest writing most of this configuration in ``~/.ssh/config``, so we abstract away all those configurations. E.g:
```
Host <hostname>: 
    StrictHostKeyChecking no
    UserKnownHostsFile /dev/null
    IdentityFile <bla-bla-id_rsa>
    ...
```

Reply:
not sure that we have permission to this folder. will try

Reply:
like the ``~/fix_uid.sh`` script, you can always reuse
please do that, we're repeating those options everywhere and it makes it unreadable